docs/recipes/isolating-mongodb-integration-tests.md

```diff
@@ -0,0 +1,86 @@
+# Setting up Ava for Isolated MongoDB Integration Tests
+
+This recipe outlines how to run disposable MongoDB databases in your Ava tests, isolated between tests.
+
+## Install MongoDB in-memory Server (MongoMem)
+In the root directory of your application, run:
+
+`npm install mongomem --save-dev`
+
+## Using MongoMem
+In your test file, import the module and run the server. 
+*Make sure to run the server at the start of your file, outside of any test cases.*
```

Comment:
Did you intend for this to be on its own line? Cause GitHub renders it right after "run the server."